css/styles.css

```
@@ -1,9 +1,331 @@
+@font-face {
+    font-family: "Inter";
+    font-style: normal;
+    font-weight: 400;
+    src: url(../fonts/Inter-400.woff2);
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "Inter";
+    font-style: normal;
+    font-weight: 700;
+    src: url(../fonts/Inter-700.woff2);
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "Inter";
+    font-style: normal;
+    font-weight: 900;
+    src: url(../fonts/Inter-900.woff2);
+    font-display: swap;
+}
+
 body {
+    font-family: Inter, sans-serif;
+    font-size: 16px;
+    line-height: 22px;
     color: #2D3440;
     background: #FEAFC3;
     margin: auto;
 }
 
+.navigation-link {
+    font-family: Inter, sans-serif;
```

```suggestion
    font-family: inherit;
```
Вместо того чтобы описывать это для каждой кнопки, лучше написать так